neurons/text/prompting/validators/core/neuron.py

```diff
@@ -550,6 +552,9 @@ def train( self ):
                     self.my_nominators = { nomin[0]: nomin[1] for nomin in delegates[0][0].nominators } if len(delegates) else {}
                     self.check_weights()
 
+                    if self.metagraph.n > self.gating_model.num_uids:

```

Comment:
is this just a place holder? cause i think it is not good if we have to wipe out the weight every time there is a new guy joining xd 

Or we can initiate a over sized linear layer at the beginning.

Reply:
yeap, but @dl-guru is looking for a better solution to this problem, this is simple patch for the issue. We shouldn't wipe out the weights everytime